src/fmt/mod.rs

```diff
@@ -1,4 +1,16 @@
+macro_rules! fmt_impl {
+    ($tr:ident, $ty:ty) => {
+        impl $tr for $ty {
+            fn fmt(&self, f: &mut Formatter<'_>) -> Result {
+                $tr::fmt(&BytesRef(self.as_ref()), f)
+            }
+        }
+    };
+}
+
+#[macro_use]
 mod debug;
+#[macro_use]
 mod hex;
```

Comment:
These modules don't export any macros for use outside of the module, so they should not be marked `#[macro_use]`.